bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -232,7 +247,8 @@ public Response startDialog(
             @QueryParam("ksId") @Parameter(description = "keywork spotter ID") @Nullable String ksId,
             @QueryParam("sttId") @Parameter(description = "Speech-to-Text ID") @Nullable String sttId,
             @QueryParam("ttsId") @Parameter(description = "Text-to-Speech ID") @Nullable String ttsId,
-            @QueryParam("hliId") @Parameter(description = "interpreter ID") @Nullable String hliId,
+            @QueryParam("hliId") @Parameter(description = "interpreter ID", deprecated = true) @Nullable String hliId,
```

Comment:
@J-N-K : is it really necessary to keep this parameter and mark it as deprecated while this API is not present in OH 3.2 ? It was recently added by me in 3.3.

Reply:
If it was not in a release, it can be removed IMO. But this only applies to the REST parameters, correct?

Reply:
It also applies to method listenAndAnswer marked as deprecated in interface VoiceManager while it is also a recently added method not present in any release. So I suggest to suppress the one currently marked as deprecated in the PR. 

But your remark about method startDialig was fully correct as this method is there since several years. So deprecating the method is the good option in that case.